xclim/indices/helpers.py

```diff
@@ -362,3 +362,38 @@ def day_lengths(
     ).assign_attrs(units="h")
 
     return day_length_hours
+
+
+def wind_speed_at_2_meters(
```

Comment:
Could we generalize this method to any height (and not only 2 m)?

Reply:
Yes, I will do this